src/w3c/headers.js

```diff
@@ -424,12 +424,15 @@ export function run(conf) {
     conf.alternateFormats && conf.alternateFormats.length > 1;
   conf.alternatesHTML =
     conf.alternateFormats &&
-    joinAnd(conf.alternateFormats, alt => {
-      let optional =
-        alt.hasOwnProperty("lang") && alt.lang ? ` hreflang='${alt.lang}'` : "";
-      optional +=
-        alt.hasOwnProperty("type") && alt.type ? ` type='${alt.type}'` : "";
-      return `<a rel='alternate' href='${alt.uri}'${optional}>${alt.label}</a>`;
+    htmlJoinAnd(conf.alternateFormats, alt => {
+      const optional = [];
+      optional.push(
+        alt.hasOwnProperty("lang") && alt.lang ? ` hreflang='${alt.lang}'` : ""
+      );
+      optional.push(
+        alt.hasOwnProperty("type") && alt.type ? ` type='${alt.type}'` : ""
+      );
+      return hyperHTML`<a rel='alternate' href='${alt.uri}'${optional}>${alt.label}</a>`;
```

Comment:
I'm not sure this will work... consider doing this:

```js
const lang = alt.hasOwnProperty("lang") && alt.lang ? alt.lang : null;
const type = alt.hasOwnProperty("type") && alt.type ? alt.type : null;
```

and then insert this in hyperHTML:

```js
hyperHTML`<a hreflang="${lang}">` // this will add `hreflang` only when `lang` is non-null.
```